code/modules/background/defines.dm

```diff
@@ -90,4 +90,8 @@
 
 #define ACCENT_DIONA "Diona"
 
-#define ACCENT_SILICON "Text-to-Speech"
\ No newline at end of file
+#define ACCENT_SILICON "Text-to-Speech"
```

Comment:
This isn't very Silicon at this point, you should search  & replace it with ACCENT_TTS